tests/MoBi.Tests/Presentation/MoleculeStartValueDTOSpecs.cs

```diff
@@ -20,26 +20,6 @@ protected override void Context()
       }
    }
 
-   public class When_validating_a_start_value_with_proposed_path_element_removed_from_end : concern_for_MoleculeStartValueDTO
```

Comment:
This test becomes invalid because it tests if the building block is valid by construction rather than through a user edit

Reply:
![image](https://user-images.githubusercontent.com/261477/212980540-e8dc153c-f322-441d-93a6-031d99b716da.png)

The test still works if triggered through a user edit